experiments/functional/backup_and_restore/setup_dependency.yml

```diff
@@ -18,47 +18,86 @@
   register: velero
   failed_when: "velero_version not in velero.stdout"
 
-- name: Installing minio s3-bucket
-  shell: kubectl apply -f velero-{{ velero_version }}-linux-amd64/examples/minio/00-minio-deployment.yaml
+- block:
 
-- name: Checking for minio pod status
-  shell: kubectl get pod -n velero -l component=minio -ojsonpath='{.items[0].status.phase}'
-  register: minio_status
-  until: "'Running' in minio_status.stdout"
-  delay: 5
-  retries: 15
+   - name: Installing minio s3-bucket
+     shell: kubectl apply -f velero-{{ velero_version }}-linux-amd64/examples/minio/00-minio-deployment.yaml
+     when: "bucket_type == 'minio'"
```

Comment:
why this when condition is required here this block itself  already has the when condition